general/administration/reverse-proxy.md

```diff
@@ -239,117 +240,42 @@ DOMAIN_NAME {
 }
 ```
 
-## LetsEncrypt with Certbot
```

Comment:
Is there any good reasons for changing the order of these sections?
As is, it makes the diff much harder to read.

Reply:
As the page currently is, I find it harder to read. The Let's Encypt section is compatible with several over RP's so it is sort of confusing to see Apache/Nginx configs again. having it at the bottom of the page helps isolate it. The entire RP page needs to be reworked to break the different sections into their own pages.

Reply:
As the page currently is, I find it harder to read. The Let's Encypt section is compatible with several over RP's so it is sort of confusing to see Apache/Nginx configs again. having it at the bottom of the page helps isolate it. The entire RP page needs to be reworked to break the different sections into their own pages.

Reply:
As the page currently is, I find it harder to read. The Let's Encypt section is compatible with several over RP's so it is sort of confusing to see Apache/Nginx configs again. having it at the bottom of the page helps isolate it. The entire RP page needs to be reworked to break the different sections into their own pages.

Reply:
I agree that the different reverse proxy solutions need their own pages.
I think that should be done in a separate pr however, as the moving of the section makes reviewing this pr more difficult than it need be.

Reply:
Reviewing this commit on its own should help (the others are cosmetic): https://github.com/jellyfin/jellyfin-docs/pull/168/commits/a0a9e4887384ab3279fe9894bd8146a9a7b15b08

See my comments below about reordering - I really think it is better in this PR than the original.